### ISSUE TITLE
CMV: The "build system" is not worth the effort

### ISSUE THREAD

**Issue by ashdar** [AshdarPartners/ScienceOven#13]
# Change My View: The build system is not worth the time and effort that it has cost this project.

## Items:
- The project is too small to reap any reward out of built help files or other automation.
- The build system seems to hamper releases via GitHub actions. The basic problem is that Publish-Module can't find the ScienceOven.psd1 file, though I have tried several experiments, each with different paths or code to log what paths are being attempted or where files might be. I suspect (most lately) that the GH Action ought to be pulling from the .\Output\ScienceOven\v0.0.x directories, but I can't find an example that helps me. Debugging GH seems to require using a third-party action, which allows tmux vis ssh, and that seems way more effort than I want to spend on this.
- A week or two ago, I spent a couple of hours debugging a problem in the Pester tests only to find that it was caused by the build system wanting to use .PSD1/PSM1 files in .\v0.0.04, which was not being updated because I didn't run a 'full' build. In short, the Pester tests could not find a new cmdlet that I added (but only after I had restarted my laptop-i must have had the module loaded outside of the tests during my prior work session because Pester was finding that new cmdlet then). I don't like running 'full builds' because they take a good deal longer. The MarkDown files and other "release artifacts" that are generated have no use to anyone, as far as I can tell.

## To sum up
There is a lot of wasted effort going into working around what the build system wants me to do, there is some non-obviousness as to what the build system wants me to do. If there is no measurable improvement in the project, why am I doing this?
